src/scrape.ts

```diff
@@ -68,6 +68,25 @@ const docsWebhooksPageForVersion = (version: string) => `https://shopify.dev/doc
 
 let warnings = 0;
 let errors = 0;
+const alphabeticallySortObjectByKeys = (obj: Object): Object => {
```

Comment:
might be worth using `safe-stable-stringify` for this so we don't have to maintain this code

Reply:
Updated to use a lib to do the stable object key sorting, although I had to add a small lodash based utility to make sort string arrays